iFixit/toolbox
# Add search box component

This adds `SearchInput` and removes `Input`, which was temporarily used as a simple search input. 
Additionally, this adds a new constant -`shadows` (src/constants/shadows.js) 🎉 

![image](https://user-images.githubusercontent.com/10774982/36743321-def209e0-1b9e-11e8-9f61-e5437df95276.png)

![image](https://user-images.githubusercontent.com/10774982/36743312-d775cce2-1b9e-11e8-8774-469a8f6aaa61.png)


## QA
- Ensure that https://deploy-preview-30--toolbox.netlify.com/ matches the screenshot above (except everywhere it says SearchBox, it should say SearchInput)
- Ensure consistency across all browsers and devices


### Notes: 
- On IE you will see this error
![image](https://user-images.githubusercontent.com/10774982/36748216-0b81c4c0-1bac-11e8-8486-8f0e51bbea09.png)

Don't worry about this. We are missing a polyfill on toolbox, but it shouldn't be an issue on iFixit. You'll notice that the same error occurs with `TextField`

- I renamed SearchBox to SearchInput, so don't be worried about that.
